.gitpod.yml

```
@@ -0,0 +1,2 @@
+tasks:
+  - init: npm install
```

```suggestion
  - init: npm ci
```

Better do avoid any potential package.json changes